client/src/components/reviews/reviewListComponents/ReviewsList.jsx

```diff
@@ -49,30 +49,30 @@ let ReviewsList = ( {productID} ) => {
     setCurrentDisplay(wholeReviewList.slice(0, reviewIndex))
   }, [reviewIndex]);
 
-  // useEffect(()=>{
-  //   // when the sorting method changes, make new GET request to get data sorted appropriately
-  //   console.log('what is the sortFilter:', sortFilter)
-  //   axios({
-  //     method: 'get',
-  //     url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/reviews?product_id=${productID}&page=1&count=50&sort=-${sortFilter}`,
-  //     headers: {
-  //       'Authorization': API_KEY
-  //     }
-  //   })
-  //     .then(({data})=>{
-  //       // console.log(`I made a reqest for new data`, data.results)
-  //       reviewArray = data.results;
-  //       setWholeReviewList(data.results);
-  //       setCurrentDisplay(reviewArray.slice(0, reviewIndex));
-  //     })
-  //     .catch((err) => {
-  //       console.log(err);
-  //     })
-  // }, [sortFilter])
+  useEffect(()=>{
+    // when the sorting method changes, make new GET request to get data sorted appropriately
+    console.log('what is the sortFilter:', sortFilter)
+    axios({
+      method: 'get',
+      url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/reviews?product_id=${productID}&page=1&count=50&sort=-${sortFilter}`,
+      headers: {
+        'Authorization': API_KEY
+      }
+    })
+      .then(({data})=>{
+        // console.log(`I made a reqest for new data`, data.results)
+        reviewArray = data.results;
+        setWholeReviewList(data.results);
+        setCurrentDisplay(reviewArray.slice(0, reviewIndex));
+      })
+      .catch((err) => {
+        console.log(err);
+      })
+  }, [sortFilter])
```

Comment:
This is another optimization thing for the backlog, but it might be good to making an array of promises and 
using Promise.all to resolve them.

Reply:
Would love to talk to you more about this. I don't really have a good grasp of how the array of promises would help here.